Blockzilla/OverlayView.swift

```diff
@@ -15,7 +15,9 @@ protocol OverlayViewDelegate: class {
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-    private let searchButton = InsetButton()
+    private var searchButtonGroup = [InsetButton]()
+    // TODO: Decide how many buttons
+    private let searchButtonNumber = 4
```

Comment:
nit: naming this searchSuggestionsCount would be more in line with swift conventions and be less ambiguous